### PR TITLE
fixed issue with everyone being considered admin on lh (garage)

### PR DESCRIPTION
**Pull request by HakonRydland**
## What type of PR is this.
1. [x] Bug
2. [ ] Change
3. [ ] Enhancement

### What have you changed and why?
Information:
    on lh it always returned true for cmdClient calls on the server( where its mostly used as most checks that need it are server side) this made everyone able to act as admin/commander on a lh session

### Please specify which Issue this PR Resolves.
closes #XXXX

### Please verify the following and ensure all checks are completed.

1. [ ] Have you loaded the mission in LAN host?
2. [ ] Have you loaded the mission on a dedicated server?

### Is further testing or are further changes required?
1. [ ] No
2. [ ] Yes (Please provide further detail below.)

### How can the changes be tested?
Steps: 

********************************************************
Notes:

### CODE REVIEW

**Comment by jaj22** [comment]
Is there a good reason that this isn't a function that returns a boolean yes/no for whether the user is allowed to override? That's how it's being used at the moment, and it'd make it a lot easier to read if it was written like that.

File: A3A/addons/Garage/Public/config.inc [hunk 1]
```diff
@@ -19,7 +19,7 @@
 HR_GRG_Prefix = "Antistasi";
 //Overwrite client
 HR_GRG_cmdClient = {
-    if ((isServer && hasInterface) || {admin owner _this > 0}) then {_this} else {theBoss}; //LAN Host or admin on DS or commander
+    if ((isServer && hasInterface && _this isEqualTo player) || {admin owner _this > 0}) then {_this} else {theBoss}; //LAN Host or admin on DS or commander
```
Comment (Bob-Murphy) [inline A3A/addons/Garage/Public/config.inc:22]: Please adapt to bool check as discussed on discord

**Review by Bob-Murphy — CHANGES_REQUESTED** [review]

**Comment by Bob-Murphy** [comment]
@jaj22 please new technical review for change to boolean

**Review by Bob-Murphy — APPROVED** [review]
LGTM - no typos found

**Review by jaj22 — APPROVED** [review]
Looks good.

**Pull request merged**